holidays/locale/fr/LC_MESSAGES/DZ.po

```diff
@@ -0,0 +1,72 @@
+# Algeria holidays fr localization.
+# Authors: Abdelkhalek Boukli Hacene <bouklihabd31@gmail.com>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.29\n"
+"POT-Creation-Date: 2023-06-28 00:13+0100\n"
+"PO-Revision-Date: 2023-09-12 15:08+0100\n"
+"Last-Translator: Abdelkhalek Boukli Hacene <bouklihabd31@gmail.com>\n"
+"Language-Team: Python Holidays localization team\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Generated-By: Lingua 4.15.0\n"
+"X-Generator: Poedit 3.3.2\n"
+
+#. Estimated label.
+#, c-format
+msgid "(تقدير*) *%s"
+msgstr "%s* (*estimé)"
+
+#. New Year's Day.
+msgid "رأس السنة الميلادية"
+msgstr "Nouvel an"
```

Comment:
As I'm not a native French speaker (not sure if you are) I don't know which one is a better option here -- https://github.com/vacanza/python-holidays/blob/06bc39f0ea66303c537a04c2e57ad184ab44d486/holidays/countries/france.py#L77

@KJhellico any suggestions?

Reply:
I'm proficient in French, both are perfectly correct. I think some holiday names should be standardized across all countries.

Reply:
>I think some holiday names should be standardized across all countries.

In this case I'd trust your knowledge here. Let's focus on this PR goals and address the French holiday names discrepancy later. I cerated https://github.com/vacanza/python-holidays/issues/1471 (low priority). Would it be okay to assign it to you as you (as far as I know) are the most proficient in French among the PH contributors.

Reply:
>Would it be okay to assign it to you as you (as far as I know) are the most proficient in French among the PH contributors.

Yes, sure.

Reply:
> Yes, sure.

I experimented w/ the organization's teams/privileges today. The bottom line is that you should find an invitation to PH contributors team in your mailbox. This will allow us to streamline the issue assigning process. This also makes your account visible at vacanza organization's page as a part of it (I believe it can be disabled if needed). I hope you'll accept the invitation, let me know if you have any questions re. this.

Thank you!

Reply:
@kasya, @mborsetti, and @PPsyrius should receive those too.

Thank you!

Reply:
@arkid15r I'm glad to be part of the team! 

I think @KJhellico's suggestions make sense for this PR.

Reply:
> @arkid15r I'm glad to be part of the team!

We always appreciate consistent and high quality contributions 👍

Reply:
@abh31000, welcome aboard! 🚀